spec/gate/indexSpec.js

```diff
@@ -89,4 +89,27 @@ describe('UserGate', function() {
       expect(gate.allows('bar@mixmax.com')).toBe(true);
     });
   });
+
+  describe('uniform sample', () => {
```

Comment:
Can you add tests of `allowsUniform`? We'd want to check the interaction of this with the list. Or, update the tests of `allows` per my comment about merging into that.

Reply:
> update the tests of allows per my comment about merging into that

https://github.com/mixmaxhq/user-gate/pull/5/commits/bbaee3d607cd7dc45dd72ebe13de72fb55c7c6d1